include/Core/Base/Mixin/Renderer.php

```diff
@@ -560,12 +560,30 @@ protected function render_delete_settings( $hide_trash = false ) {
 	 * Render limit settings.
 	 *
 	 * @param string $item_type Item Type to be displayed in label.
+	 * @param string $remove    Toggle labels between delete and remove(Used in multisites).
+	 *
+	 * @since 6.1.0 Added $remove param.
 	 */
-	protected function render_limit_settings( $item_type = '' ) {
+	protected function render_limit_settings( $item_type = '', $remove = false ) {
 		if ( empty( $item_type ) ) {
 			$item_type = $this->item_type;
 		}
-		bd_render_limit_settings( $this->field_slug, $item_type );
+		?>
+		<tr>
+			<td scope="row">
+				<input name="smbd_<?php echo $this->field_slug; ?>_limit" id="smbd_<?php echo $this->field_slug; ?>_limit" value="true" type="checkbox">
+			</td>
+			<td>
+				<?php if ( $remove ) : ?>
+				<label for="smbd_<?php echo $this->field_slug; ?>_limit"><?php _e( 'Only remove first ', 'bulk-delete' ); ?></label>
+				<?php else : ?>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method render_cron_settings uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/app/BulkWP/bulk-delete/pullRequest?prid=4259873)